amazon_payments/api.py

```diff
@@ -193,7 +193,7 @@ def get_amazon_order_details(self, billing_agreement_id, access_token,
         return True, amazon_order_details
 
     def create_order_reference_id(self, billing_agreement_id, order_amount,
-                                  currency, **kwargs):
+                                  currency, order_id='N/A', **kwargs):
```

Comment:
@sjolicoeur Can we just pass the `order` in instead of the pk? I could see this argument list getting to be a bit out of hand. Also, is it available in scope? Maybe `self.order` already exists?